tests/metrics/genai/test_genai_metrics.py

```diff
@@ -1034,3 +1051,98 @@ def test_make_genai_metric_without_example():
+    expected_metric_metadata = {"metadata_field": "metadata_value"}
```

Comment:
we could define this above the caller and assign it to the func and then validate that the dict is preserved, right?